WorkflowGuidelines.md

```diff
@@ -79,6 +79,32 @@
       let savedData = System.IO.File.ReadAllText saveFilePath.FullName
       ```
 
+    * Discarding exceptions:
+
+        Discarding exceptions is a bad practice because it can lead to unexpected behavior and bugs. It's better to handle the exception in some way, or at least log them to help with debugging if something unex
+    
+        Example:
+        ```f#
+        let func zipPath =
+            try
+                ZipFile.Open(zipPath.ToString(), ZipArchiveMode.Read) |> ignore
+            with
+            | ex -> 
+                ()
+        ```
+        ```f#
+        let func zipPath =
+            try
+                ZipFile.Open(zipPath.ToString(), ZipArchiveMode.Read) |> ignore
+            with
+            | :? InvalidDataException -> 
+                printfn $"Not a zip file %s{zipPath}"
+            | :? FileNotFoundException -> 
+                printfn $"File not found %s{zipPath}"
+            | ex -> 
+                raise ex
```

Comment:
@parhamsaremi this is another bad practice: if you are raising the same exception you have to use `reraise`